games-strategy/julius/julius-1.3.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="An open source re-implementation of Caesar III"
+HOMEPAGE="https://github.com/bvschaik/julius"
+SRC_URI="https://github.com/bvschaik/julius/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	media-libs/libpng
+	media-libs/libsdl2
+	media-libs/sdl2-mixer
+"
+RDEPEND=""
```

Comment:
Ah, libsdl is needed only during build time, but not during runtime?

Reply:
Yes, just at build time.

Reply:
It doesn't use libsdl.so?

Reply:
Why did you resolve this without fixing?